Resources/views/chart/visualizationChart.html.twig

```diff
@@ -1,7 +1,32 @@
-var data{{ id }} = new google.visualization.DataTable({{ data | json_encode | replace({"\"[new Date]": 'new Date', "[new Date]\"": ""}) | raw }});
+{% if chartType == 'Calendar' or chartType == 'Timeline' %}
+    {% set data = data | json_encode | replace({"\"new Date[[[": 'new Date(', "new Date\"": ""}) | replace({"]]]\"": ')', "]]]\"": ")"}) | replace({"id\":\"role_tooltip": "role\": \"tooltip", "id\":\"role_tooltip": "role\": \"tooltip"}) %}
```

Comment:
Do you think there is a better way to deal with the "date" transformation you are doing here?